Library/Homebrew/test/cask/blacklist_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+describe Cask::Blacklist, :cask do
+  describe "::blacklisted_reason" do
```

Comment:
I'd suggest making this similar to https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/missing_formula_spec.rb#L12-L35.